### PR TITLE
Fix readme

### PR DESCRIPTION
**Pull request by BenoitClaveau**
- [ ] CHANGELOG.md has been updated

### CODE REVIEW

**Comment by nephiw** [comment]
I signed it!

On Dec 20, 2016 2:33 AM, "googlebot" <notifications@github.com> wrote:

Thanks for your pull request. It looks like this may be your first
contribution to a Google open source project. Before we can look at your
pull request, you'll need to sign a Contributor License Agreement (CLA).

📝 *Please visit https://cla.developers.google.com/
<https://cla.developers.google.com/> to sign.*

Once you've signed, please reply here (e.g. I signed it!) and we'll verify.
Thanks.
------------------------------

   - If you've already signed a CLA, it's possible we don't have your
   GitHub username or you're using a different email address. Check your
   existing CLA data <https://cla.developers.google.com/clas> and verify
   that your email is set on your git commits
   <https://help.github.com/articles/setting-your-email-in-git/>.
   - If you signed the CLA as a corporation, please let us know the
   company's name.

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/Polymer/polymer-build/pull/91#issuecomment-268196970>,
or mute the thread

File: README.md [hunk 1]
```diff
@@ -111,11 +111,11 @@ const htmlMinifier = require('gulp-html-minifier');
 const mergeStream = require('merge-stream');
 
 const sourcesStream = polymerProject.sources()
```
Comment (FredKSchott) [inline README.md]: missed one here

File: README.md [hunk 2]
```diff
@@ -137,7 +137,7 @@ const generateServiceWorker = require('polymer-build').generateServiceWorker;
 
 generateServiceWorker({
   buildRoot: 'build/',
-  project: polymerProject,
+  project: project,
   bundled: true // set if `polymerProject.bundler` was used
```
Comment (FredKSchott) [inline README.md]: missed on here (in the comment)

**Comment by FredKSchott** [comment]
@nephiw I don't believe you can sign the CLA for others, @BenoitClaveau will have to sign the CLA himself

**Comment by nephiw** [comment]
I missunderstood, thanks for informing me.

**Comment by FredKSchott** [comment]
@BenoitClaveau could you take another look at this? Would love to get this merged

**Comment by FredKSchott** [comment]
Fixed in ed50257. Thanks for the PR @BenoitClaveau

**Pull request closed without merging**